### PR TITLE
Update aws-ebs-csi-driver

### PR DESCRIPTION
**Pull request by ialidzhikov**
/area storage
/platform aws

**Release note**:

```other user
The following image is updated:
- k8s.gcr.io/provider-aws/aws-ebs-csi-driver: v0.10.1 -> v1.1.0 (see [CHANGELOG](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/CHANGELOG-0.x.md#v110))
```

### CODE REVIEW

File: charts/images.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ images:
 - name: csi-driver
   sourceRepository: github.com/kubernetes-sigs/aws-ebs-csi-driver
   repository: k8s.gcr.io/provider-aws/aws-ebs-csi-driver
-  tag: "v0.10.1"
+  tag: "v1.1.0"
```
Comment (vpnachev) [inline charts/images.yaml:52]: Can we wait for the [next](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/924#issuecomment-859771242) release planned next week?

Comment (ialidzhikov) [reply charts/images.yaml:52]: Okay, let's then wait for v1.1.1.

/hold

Comment (ialidzhikov) [reply charts/images.yaml:52]: There is no activity in the upstream for v1.1.1.

/hold
/remove lifecycle/stale

Comment (ialidzhikov) [reply charts/images.yaml:52]: /unhold

**Review by vpnachev — APPROVED** [review]
/lgtm

**Pull request merged**